tiny-project/src/components/wood-api-detail.tsx

```diff
@@ -0,0 +1,48 @@
+import { useParams } from 'react-router-dom';
+import {useEffect, useState} from 'react';
+import api from '../api/api';
+
+interface WoodDetail {
+    'name': string,
+    'description': string,
+    'id': number,
+    'image': string,
+    'location': string,
+    'slug': string,
+    'lat': number,
+    'long': number,
+    'url': string
+}
+
+export default function WoodApiDetail() {
+    const url = 'https://tree-nation.com/api/projects'
+    let { id } = useParams();
+
+    useEffect(() => {
+        getData();
+    },[])
+
+    const [data, setData] = useState<WoodDetail>({
```

Comment:
set initial state as undefined and use conditional rendering later in return part